packages/StepTracker/StepTracker.jsx

```diff
@@ -1,95 +1,70 @@
-import React, { Component } from 'react'
+import React from 'react'
 import PropTypes from 'prop-types'
-import classNames from 'classnames'
+
+import Text from '@tds/core-text'
+import Flexbox from '../../shared/components/Flexbox/Flexbox'
```

Comment:
Should we be marking Flexbox as a private package now and declaring it as a dev dependency? I think thats what we have been doing as we are touching the shared packages.

Reply:
Ahhhh yes, you are correct. I thought you were adding a prop value to the Flexbox component so that we could do `<Flexbox justifyContent="center">`